src/test/java/integration/UserControllerIT.java

```diff
@@ -48,14 +50,14 @@ public void createUser() throws Exception {
                         statusCode(SC_OK).
                 extract()
                         .response();
-        long timeFinished = currentTimeMillis();
+        LocalDateTime timeFinished = now();
 
         User createdUser = userService.getEntityByToken(UUID.fromString(response.path("token")));
-        long timeCreated = createdUser.getCreated().getTime();
-        long timeModified = createdUser.getLastModified().getTime();
+        LocalDateTime timeCreated = createdUser.getCreated();
+        LocalDateTime timeModified = createdUser.getLastModified();
 
-        assertTrue(WRONG_CREATED_MESSAGE, timeFinished >= timeCreated && timeCreated >= timeStarted);
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_CREATED_MESSAGE, timeFinished.isAfter(timeCreated) && timeCreated.isAfter(timeStarted));
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).